Bynder/Models/Media.cs

```diff
@@ -131,5 +131,18 @@ public class Media
         /// </summary>
         [JsonProperty("tags")]
         public IList<string> Tags { get; set; }
+
+        /// <summary>
+        /// Indicates if the media item is public
+        /// </summary>
+        [JsonProperty("isPublic")]
+        public bool IsPublic { get; set; }
```

Comment:
Due to our API returning 1/0 we should use BooleanJsonConverter to convert that to boolean as we do in
https://github.com/Bynder/bynder-c-sharp-sdk/blob/1e72e880a6924c5c4c4965a3ed84956f159d6c03/Bynder/Models/MetaProperty.cs#L60